client/qiskit_serverless/core/clients/local_client.py

```diff
@@ -41,7 +41,6 @@
 
 from qiskit_serverless.core.constants import (
     OT_PROGRAM_NAME,
-    ENV_JOB_ARGUMENTS,
```

Comment:
Probably this is for another PR because we will need to take a look but I think we can remove all the references for `ENV_JOB_ARGUMENTS` in the `client`. In the `gateway` it will require a bit more analysis but I think so too.

Reply:
I didnt removed it because of `ray_client` that it is using it and I don't know if it will raise any problem.

Reply:
Absolutely, for another PR. I will create an issue annotating this improvement.